src/dune_project.ml

```diff
@@ -105,7 +105,7 @@ end = struct
              |> List.tl
              |> String.concat ~sep:"/")
         in
-        if not (Path.is_local p) then invalid s;
+        if not (Path.is_managed p) then invalid s;
```

Comment:
@diml I think this check and the one above can perhaps be tightened a little. E.g. `Path.is_in_source_tree` instead.

Reply:
Indeed